### PR TITLE
flutter: from v1.5.4-hotfix.2-stable to v1.12.13+hotfix.7

### PR DESCRIPTION
**Pull request by alvr**
Update to latest version `v1.12.13+hotfix.7` from `v1.5.4-hotfix.2-stable`

### CODE REVIEW

**Comment by omerakgoz34** [comment]
@alvr you must be update "version" too :)
"version": "1.5.4-hotfix.2"
"version": "1.12.13+hotfix.5"

**Comment by alvr** [comment]
@omerakgoz34 done!

**Comment by NoDocCat** [comment]
@r15ch13 
Please merge this PR as soon as possible. flutter@1.5.4 is too old

File: bucket/flutter.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
     "homepage": "https://flutter.dev/",
-    "version": "1.5.4-hotfix.2",
+    "version": "v1.12.13-hotfix.5",
```
[the remark this answers is not in the document]
Comment (alvr) [reply bucket/flutter.json]: Done. Sorry for the delay.

**Pull request closed without merging**